app/src/test/java/org/simple/clinic/summary/prescribeddrugs/DrugSummaryUiControllerTest.kt

```diff
@@ -64,19 +69,25 @@ class DrugSummaryUiControllerTest {
   }
 
   @Test
-  fun `when update medicines is clicked then updated prescription screen should be shown`() {
+  fun `when update medicines is clicked and facility is not switched then updated prescription screen should be shown`() {
```

Comment:
Since the controller does not directly care about the facility being switched anymore, I think we can leave the test name intact. What do you think?